test/e2e/specs/editor/various/block-mover.spec.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'block mover', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'should show block mover when more than one block exists', async ( {
+		editor,
+		page,
+	} ) => {
+		// Create a two blocks on the page.
+		await page.click( '.block-editor-default-block-appender' );
+		await page.keyboard.type( 'First Paragraph' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( 'Second Paragraph' );
+
+		// Select a block so the block mover is rendered.
+		await page.focus( 'text=First Paragraph' );
+		await editor.showBlockToolbar();
+		const count = await page.$$eval(
+			'.block-editor-block-mover',
+			( el ) => el.length
+		);
+		expect( count ).toBe( 1 );
```

Comment:
It'd be good to take advantage of Playwright's locator system here, and use the role selector:
```js
const moveDownButton = page.locator( 'role=toolbar[name="Block tools"i] >> role=button[name="Move down"i]' );
const moveUpButton = page.locator( 'role=toolbar[name="Block tools"i] >> role=button[name="Move up"i]' );
await expect( moveDownButton ).toBeVisible();
await expect( moveUpButton ).toBeVisible();
```

Reply:
Worked on the feedbacks